flow-typed/hapi.js

```diff
@@ -192,10 +235,21 @@ declare type HapiResponseToolkit = {
 	realm: any,
 	request: HapiRequest,
 	authenticated: (data: HapiAuthenticationData) => Object,
-	entity: (options?: {etag?: string, modified?: string, vary?: boolean}) => HapiResponseObject | void,
+	entity: (options?: {
+		etag?: string,
+		modified?: string,
+		vary?: boolean,
+	}) => HapiResponseObject | void,
 	redirect: (uri?: string) => HapiResponseObject,
 	response: (value?: HapiResponseValue) => HapiResponseObject,
-	state: (key: string, value: string | Object, options?: HapiServerStateCookieOptions) => HapiResponseObject,
+	state: (
+		key: string,
+		value: string | Object,
+		options?: HapiServerStateCookieOptions
+	) => HapiResponseObject,
 	unauthenticated: (error: Error, data?: HapiAuthenticationData) => void,
-	unstate: (key: string, options?: HapiServerStateCookieOptions) => HapiResponseObject,
-};
\ No newline at end of file
+	unstate: (
+		key: string,
+		options?: HapiServerStateCookieOptions
+	) => HapiResponseObject,
```

Comment:
Did anything change in this file? It looks like just formatting but if not, please call out what changed.

Reply:
I think I added something here, saved it, and then later removed it, so just the Prettier formatting changes were left - I'll see if I can flip it back to the `master` version so it doesn't show in the diff.